### PR TITLE
[58] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by faithakinbobola**
## Task: Develop an E-commerce Store Database`
- `Developed the database for my E-commerce store`

### CODE REVIEW

File: submissions/akinbobolaFaith/database/init.sql [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+(2, 'No 1, Clinton Close', 'Ibadan', 'Oyo', 00130, 'Nigeria', '08012345601', now()),
+(3, 'No 3, Mobayode Street', 'Abeokuta', 'Ogun', 00120, 'Nigeria', '08012345602', now()),
+(4, 'No 7, Hamilton Avenue', 'Awka', 'Anambra', 00014, 'Nigeria', '08012345603', now()),
+(5, 'No 12, Paragon Close', 'Enugu', 'Enugu', 00173, 'Nigeria', '08012345604', now());
+ 
+ 
+/*********************CATEGORIES TABLE*************************/
+CREATE TABLE categories (
+    id INT NOT NULL AUTO_INCREMENT,
+	products_id INT NOT NULL,
```
Comment (adaugochi) [inline submissions/akinbobolaFaith/database/init.sql:70]: kindly read the instructions again thoroughly. you were told `each product belongs to a category` and not `each category belongs to a product`

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/akinbobolaFaith/database/init.sql [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+
+/*********************CUSTOMERS TABLE*************************/
+CREATE TABLE customers (
+	id INT NOT NULL AUTO_INCREMENT,
+	firstname VARCHAR(50) NOT NULL,
+	lastname VARCHAR(50) NOT NULL,
+	email_address VARCHAR(100) NOT NULL,
+	password VARCHAR(50) NOT NULL,
+	created_by DATETIME,
```
Comment (adaugochi) [inline submissions/akinbobolaFaith/database/init.sql:69]: I think this should be `created_at`

File: submissions/akinbobolaFaith/database/init.sql [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+	PRIMARY KEY (id)
+);
+
+INSERT INTO customers (firstname, lastname, email_address, password, created_by)
```
Comment (adaugochi) [inline submissions/akinbobolaFaith/database/init.sql:73]: same here

File: submissions/akinbobolaFaith/database/init.sql [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+(4, 'No 7, Hamilton Avenue', 'Awka', 'Anambra', 00014, 'Nigeria', '08012345603', now()),
+(5, 'No 12, Paragon Close', 'Enugu', 'Enugu', 00173, 'Nigeria', '08012345604', now());
+ 
+ 
+/*********************ORDERS TABLE*************************/	
+CREATE TABLE orders (
+	id INT NOT NULL AUTO_INCREMENT,
+	customers_id INT NOT NULL,
+	amount DECIMAL(10,2) NOT NULL,
+	date VARCHAR(50) NOT NULL,
```
Comment (adaugochi) [inline submissions/akinbobolaFaith/database/init.sql:110]: `order_date` is better than just `date`

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/akinbobolaFaith/database/init.sql [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+	created_at DATETIME,
+	PRIMARY KEY (id),
+    FOREIGN KEY (category_id) REFERENCES categories (id),
+    FOREIGN KEY (admin_id) REFERENCES admins (id)
+);
+
+
+INSERT INTO products (name, description, image, unit_price, category_id, admin_id, stock_level, status, created_at)
+VALUES('R L Fit Oxford Button-Down Pony Logo Shirt-Skyblue', 'Ralph Lauren is the prinnacle of style and design. This classic fit oxford shirt is the perfect way to stay classy. A buttondown classic fit polo which is 100% cotton', 'https://C:\Users\Dell\Desktop\Dufuna-CodeCamp20\submissions\akinbobolaFaith\html\image\img_0367_1.jpg', 14000.00, 2, 4, 180, 'in stock', now()),
+("Skechers Men's Blaine Orsen Ankle Boot", 'Approx. 1'' heel height, Approx. 5'' boot shaft height, Cushioned footbed, Dual side goring Fabric lining', 'https://C:\Users\Dell\Desktop\Dufuna-CodeCamp20\submissions\akinbobolaFaith\html\image\Skechers-Men-Blaine-Orsen-Ankle-Boot.jpg', 22000.00, 3, 1, 50, 'low stock', now()),
```
Comment (adaugochi) [inline submissions/akinbobolaFaith/database/init.sql:56]: kindly use online image URLs rather than this absolute path you are using

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/akinbobolaFaith/css/contactForm.css [hunk 6]
```diff
@@ -8,8 +8,8 @@ body {
     box-sizing: border-box;
 }
 
-nav a[href*="*"] {
-     font-size: 31px;
+nav a[href="#"] {
```
Comment (adaugochi) [inline submissions/akinbobolaFaith/css/contactForm.css:11]: kindly give this a class name. Imagine you now add a link, this styling will be affected and thus you need to update it again. Same goes for others

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/akinbobolaFaith/database/init.sql (hunk 2 above)
Comment (adaugochi) [reply submissions/akinbobolaFaith/database/init.sql:69]: You haven't fix this

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/akinbobolaFaith/database/init.sql (hunk 4 above)
Comment (adaugochi) [reply submissions/akinbobolaFaith/database/init.sql:112]: and this too

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — CHANGES_REQUESTED** [review]
What is this file that is part of your PR 
image

**Comment by faithakinbobola** [comment]
I don't know. I didn't touch any shortcut file. I don't know how it came about

**Review by adaugochi — CHANGES_REQUESTED** [review]
We will look into the matter on Monday via call

**Review by adaugochi — APPROVED** [review]

**Pull request merged**